client/src/app/shared/hooks/table-controls/filtering/getFilterHubRequestParams.ts

```diff
@@ -78,6 +85,8 @@ export const serializeFilterForHub = (filter: HubFilter): string => {
   const joinedValue =
     typeof value === "string"
       ? wrapInQuotesAndEscape(value)
+      : typeof value === "number"
+      ? value
```

Comment:
You probably need it wrapped in quotes here as well, so you could either adapt the `wrapInQuotesAndEscape` function or just do `"${value}"` here.